api/README.md

```diff
@@ -291,9 +291,9 @@ HTTP URL Arguments:
 

```

Comment:
@Any new environment variables for the API need to be added to this list and provided a short description on what they do.

Reply:
Addressed. I did my best on the contact support menu variables.

Reply:
Yeah we didn't catch that on the pull request that included those. I made a small change but this looks great. Thanks for doing this.